packages/astro/test/fixtures/astro-css-bundling/src/pages/preload.astro

```diff
@@ -0,0 +1,14 @@
+---
+import Nav from '../components/Nav.astro';
+---
+
+<html>
+  <head>
+    <link rel="preload" href="../css/page-preload.css" />
```

Comment:
Could you add a test for whether preload deduping works? Otherwise, lgtm!

Reply:
Sure, tests for merging preloads added.